.github/workflows/gutenberg-packages-update.yml

```diff
@@ -113,8 +113,8 @@ jobs:
 
       - name: Configure git user
         run: |
-          git config user.email "pierregordon@protonmail.com"
-          git config user.name "Pierre Gordon"
+          git config user.name ${{ secrets.GIT_USER }}
+          git config user.email ${{ secrets.GIT_USER_EMAIL }}
```

Comment:
Who would it be in this case?

Reply:
Will be using a different user. Shared the username on slack.